native/CMakeLists.txt

```diff
@@ -1827,6 +1961,17 @@ if(USE_MIDDLEWARE)
 endif()
 
 ######## manual
+
+if(NOT OPENHARMONY)
+    cocos_source_files(
+	    NO_WERROR    cocos/bindings/manual/jsb_network_manual.cpp
+	                 cocos/bindings/manual/jsb_network_manual.h
+	    NO_WERROR    cocos/bindings/manual/jsb_xmlhttprequest.cpp
+	                 cocos/bindings/manual/jsb_xmlhttprequest.h
+
+    )
+endif()
```

Comment:
We need a new switch `USE_NETWORK`, enabled by default, disable in OPENHARMONY
```cmake
if(OPENHARMONY)
 set(USE_NETWORK OFF)
endif()
```

Reply:
It's temporary at the moment and will open later

Reply:
OK